requirements.txt

```diff
@@ -1,5 +1,5 @@
 Pillow>=2.3.0,<=3.1.2
-numpy>=1.8.1,<=1.11.0
+numpy>1.8.1,<=1.11.0
```

Comment:
Why this change? Looks like this would break our 14.04 packaging:
https://packages.ubuntu.com/trusty/python-numpy